lib/domain/plan.js

```diff
@@ -1,9 +1,30 @@
 export default class Plan {
-  constructor({ id, created, firstName, lastName, systemIds }) {
+  constructor({ id, firstName, lastName, systemIds }) {
     this.id = id;
-    this.created = created;
+    this.created = new Date(Date.now()).toISOString();
     this.firstName = firstName;
+    this.goal = null;
```

Comment:
In the future are we going to support multiple goals? If so, maybe we go for an array of 1 object for now to save a bunch of hassle switching everything later on?

Reply:
Yes we will be supporting multiple goals, but Chris defined these other goals as "sub-goals", ie there is a master goal. Kinda unsure how this is gonna look, I think FG are too

Reply:
Cool ok, makes sense then.